worlds/sc2/docs/setup_fr.md

```diff
@@ -0,0 +1,214 @@
+
+Un fichier dans le format *yaml* est utilisé pour communiquer à Archipelago comment vous voulez que votre jeu soit 
+*randomized*. 
+Ce dernier est nécessaire même si vous voulez utiliser les options par défaut. 
+L'approche usuelle pour générer un *multiworld* consiste à avoir un fichier *yaml* par monde.
+
+Il y a trois approches pour obtenir un fichier *yaml* pour *StarCraft 2 Randomizer*:
+* Vous pouvez aller à la page [*Player options*](/games/Starcraft%202/player-options) qui vous permet de définir vos 
+choix via une interface graphique et ensuite télécharger le *yaml* correspondant à ces choix.
+* Vous pouvez obtenir le modèle de base en le téléchargeant à la page 
+[*Player options*](/games/Starcraft%202/player-options) ou en cliquant sur *Generate template* après avoir éxécuté le 
+*Launcher* d'Archipelago (i.e., `ArchipelagoLauncher.exe`). Ce modèle de base inclut une description pour chacune des 
+options et vous n'avez qu'à modifier les options dans un éditeur de texte de votre choix.
+* Vous pouvez demander à quelqu'un d'autre de partager un de ces fichiers *yaml* pour l'utiliser ou l'ajuster à vos 
+préférences.
+
+Prenez soin de vous rappeler du nom de joueur que vous avez inscrit dans la page à options ou dans le fichier *yaml* 
+puisque vous en aurez besoin pour vous connecter à votre monde!
+
+Notez que la page *Player options* ne permet pas de définir certaines des options avancées, e.g., l'exclusion de 
+certaines unités ou de leurs améliorations. 
+Utilisez la page [*Weighted Options*](/weighted-options) pour avoir accès à ces dernières.
+
+Si vous désirez des informations et/ou instructions générales sur l'utilisation d'un fichier *yaml* pour Archipelago, 
+veuillez consulter [*Creating a YAML*](/tutorial/Archipelago/setup/en#creating-a-yaml).
+
+### Questions récurrentes à propos du fichier *yaml*
+#### Comment est-ce que je sais que mon *yaml* est bien défini?
+
+La manière la plus simple de valider votre *yaml* est d'utiliser le 
+[système de validation](/check) du site web.
+
+Vous pouvez aussi le tester en tentant de générer un *multiworld* avec votre *yaml*.
+Pour faire ça, sauvegardez votre *yaml* dans le dossier `Players/` de votre installation d'Archipelago et exécutez 
+`ArchipelagoGenerate.exe`. 
+Si votre *yaml* est bien défini, vous devriez voir un nouveau fichier, avec l'extension `.zip`, apparaître dans le 
+dossier `output/` de votre installation d'Archipelago.
+Il est recommandé de lancer `ArchipelagoGenerate.exe` via un terminal afin que vous puissiez voir les messages générés 
+par le logiciel, ce qui va inclure toutes erreurs qui ont eu lieu et le nom de fichier généré.
+Si vous n'appréciez pas le fait d'utiliser un terminal, vous pouvez aussi regarder le fichier *log* qui va être produit 
+dans le dossier `logs/`.
+
+#### À quoi sert l'option *Progression Balancing*?
+
+Pour un monde *StarCraft 2* seule, cette option ne fait rien. 
```

Comment:
This entire paragraph feels very "robotic". It should be something like : 

"Pour *Starcraft 2*, cette option ne fait pas grand-chose.
Il s'agit d'une option d'Archipelago permettant d'équilibrer la progression des mondes en interchangeant les *items* dans les *sphères*.  Cependant, *Starcraft 2* est beaucoup plus permissif en termes d'*items* qui permettent de progresser, ce réglage à donc peu d'influence sur la progression. Vu qu'il augmente le temps de génération d'un *MultiWorld*, nous recommandons de le désactiver pour *Starcraft 2*.

Reply:
Applied except:
- Kept the sentence that describe what the option odes 
- Kept the fact that deactivation is by using the zero value
- Explicited that "sur la progression." with "sur la progression Starcraft 2."

Reply:
The reviewer agreed after a discussion in private messages.